base-notebook/Dockerfile

```diff
@@ -70,8 +70,9 @@ RUN sed -re "s/c.ServerApp/c.NotebookApp/g" \
 # This healtcheck works well for `lab`, `notebook`, `nbclassic`, `server` and `retro` jupyter commands
 # https://github.com/jupyter/docker-stacks/issues/915#issuecomment-1068528799
 HEALTHCHECK  --interval=5s --timeout=3s --start-period=5s --retries=3 \
-    CMD wget -O- --no-verbose --tries=1 --no-check-certificate \
-    http${GEN_CERT:+s}://localhost:${JUPYTER_PORT}${JUPYTERHUB_SERVICE_PREFIX:-/}api || exit 1
+    CMD cat /home/${NB_USER}/.local/share/jupyter/runtime/*.json | \
```

Comment:
```suggestion
    CMD cat /home/"${NB_USER}"/.local/share/jupyter/runtime/*.json | \
```

Reply:
Is this to handle the case where "NB_USER" contains whitespace? I'm wondering if we need to quote the entire path `"/home/${NB_USER}/.local/share/jupyter/runtime/*.json"` instead..?

Reply:
> Is this to handle the case where "NB_USER" contains whitespace?

Yes, though it is not well tested 😂 
Abd we try to keep this style where we use variables with possible space, so it should probably work fine.

> I'm wondering if we need to quote the entire path "/home/${NB_USER}/.local/share/jupyter/runtime/*.json" instead..?

No, if you quote the entire string, `cat` is gonna think you need one specific file (and it won't find it).
So, it should be done the way I wrote it.

Reply:
Actually I tried to test the current implementation without quoting NB_USER and setting NB_USER to have space and it somehow passed the test 🤔 I committed this anyway though